FashionConnect_api/routes/users.js

```diff
@@ -66,15 +66,14 @@ router.post('/users/login', async (req, res) => {
     // Return the user data in the response
     res.json({ user });
   } catch (error) {
-    console.error(error);
     res.status(500).json({ error: 'Server error' });
   }
 });
 
 //routes for profile changes
 
 router.post('/users/profile', async (req, res) => {
-  const { user, First_Name, Last_Name, username } = req.body;
+  const { First_Name, Last_Name, username } = req.body;
```

Comment:
Alright.
I thought to keep it that way since that's how they are defined in my Database schema.
However, I will be consistent  with my casing from now on.